apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.html

```diff
@@ -43,7 +46,7 @@
     </p>
 
     <div class="buttons-container">
-      @if (!file) {
+      @if (!file()) {
```

Comment:
Switch the if/else around. Then you can do the following:
```
@if (file(); as f) {
  <hra-delete-file-button [fileName]="f.name" ...>
  ...
} @else {
  ...
}
```
This way you don't need the non-null `!` assertion